keyboards/divergetm2/rules.mk

```diff
@@ -0,0 +1,76 @@
+SRC += ../lets_split/matrix.c \
```

Comment:
Please don't refer to other keyboard's files here. It's not a good idea long term.  

Also, you should actually use the new Split Common code here.  
Eg, remove this section, and see below.